index.html

```diff
@@ -17,6 +17,11 @@
         </div>
         <nav class="menu">
             <a href="#"> <img src="./images/Union.png" alt="menu-icon" class="hamberger-menu"></a>
+            <ul class="desktop-menu">
+                <li><a href="" class="desktop-menu-item" id="porfolio">portfolio</a></li>
+                <li><a href="" class="desktop-menu-item" id="about">About</a></li>
+                <li><a href="" class="desktop-menu-item" id="contact">contact</a></li>
+            </ul>
```

Comment:
You did a good job working on this section! 🚀 However, there are some things that could make your design even better:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/213547815-6156c5da-e964-46cd-ba90-3e26c31f13df.png) | ![image](https://user-images.githubusercontent.com/77069011/213547755-f47e00e8-9f93-4976-840d-0bde49f18e8f.png) |


- [ ] 🚧 Please make sure to capitalize the first letter of your navigation links, this will provide consistency in your design, as well as being visually more comfortable. 🔧 

- [ ] 🚧 Please increase the font size of the paragraph. According to the FIGMA template, it should be `20px` instead of the current `16px`. 🔧 

- [ ] 🚧 You're having a problem with your mobile styles, the navigation links are not supposed to be displayed on mobile sizes. Please fix that! 🔧 

![image](https://user-images.githubusercontent.com/77069011/213548795-badc018b-6501-4340-a7eb-25fee4347211.png)